pages/about.vue

```diff
@@ -1,102 +1,132 @@
+                {{ $t("about.paragraph2") }}
+            </p>
+        </div>
+        <div
+            data-testid="members-header-container"
+            id="members-header-container"
+            class="flex justify-center items-center"
+        >
+            <div
+                data-testid="members-header"
+                id="members-header"
+                class="flex w-4/5 my-14"
+            >
+                <div
+                    class="flex-1 border-currentColor/70 border self-center"
+                ></div>
+                <div
+                    class="text-primary/80 text-xl font-semibold mx-4 p-2 whitespace-nowrap"
+                >
+                    {{ $t("about.members") }}
+                </div>
+                <div
+                    class="flex-1 border-currentColor/70 border self-center"
+                ></div>
             </div>
-            <div class="members-list flex flex-row" :key="index" v-for="(member, index) in members">
-                <div class="member flex flex-col">
-                    <img :href="member.avatarImg" />
-                    <div class="member-name text-xl font-bold">{{ member.name }}</div>
-                    <div class="member-title text-md text-primary-text-muted">{{ member.title }}</div>
-                    <div class="member-social flex flex-row">
-                        <a :href="member.linkedInUrl" target="_blank" rel="noopener noreferrer">
-                            <svg role="img" alt="linkedin icon" title="linkedin icon"
-                                class="social-icon w-12 h-12 mr-1 fill-primary-text-muted">
-                                <use xlink:href="../assets/images/social-linkedin.svg#social-linkedin-svg" />
-                            </svg>
-                        </a>
-                        <a :href="member.githubUrl" target="_blank" rel="noopener noreferrer">
-                            <svg role="img" alt="github icon" title="github icon"
-                                class="social-icon w-12 h-12 mr-1 fill-primary-text-muted">
-                                <use xlink:href="../assets/images/social-github.svg#social-github-svg" />
-                            </svg>
-                        </a>
+        </div>
+        <div
+            data-testid="members-container"
+            id="members-container"
+            class="flex justify-center items-center mx-4"
+        >
+            <div
+                data-testid="members"
+                id="members"
+                class="flex flex-wrap justify-between items-start w-4/5 mx-4 md:flex-wrap mx-32"
+            >
+                <div
+                    data-testid="member"
+                    class="members-list flex flex-col justify-center p-4 md:flex-row basis-1/3"
+                    :key="index"
+                    v-for="(member, index) in data.members"
+                >
+                    <div class="member flex flex-col items-center text-center">
```

Comment:
nit: you could make a separate memberComponent here

Reply:
Are you referring to the whole container or it should loop through a component that is a card that takes the `member` and `index` as props?

Reply:
the latter where you can have a member card. that you loop through. 
You can check out `SearchResultsList` and `SearchResultListItem` as an example 👍